cap-2.x/aks/providers.tf

```diff
@@ -3,7 +3,7 @@ terraform {
 }
 
 provider "azurerm" {
-  version = "2.32.0"
+  version = "2.36.0"
```

Comment:
Is v2.36 a new requirement? v2.32 is the currently packaged version:

https://build.opensuse.org/package/show/systemsmanagement:terraform/terraform-provider-azurerm

Reply:
No, it's not a requirement. Changed back to 2.32.0